src/jaxsim/parsers/rod/utils.py

```diff
@@ -206,3 +217,62 @@ def fibonacci_sphere(samples: int) -> npt.NDArray:
     return descriptions.SphereCollision(
         collidable_points=collidable_points, center=center_wrt_link
     )
+
+
+def create_mesh_collision(
+    collision: rod.Collision,
+    link_description: descriptions.LinkDescription,
+    mesh_uri: str,
+    method: MeshMappingMethods = MeshMappingMethods.VertexExtraction,
+    nsamples: int = 1000,
+) -> descriptions.MeshCollision:
+    file_obj = resolve_local_uri(uri=collision.geometry.mesh.uri)
+    try:
+        _file_type = str(file_obj).split(".")[-1]
+    except ValueError as e:
+        raise e(f"Failed to get file type from {file_obj}")
```

Comment:
If I recall, trimesh already fails if it receives an extension that is not supported. Can you just pass to trimesh `file_type=pathlib.Path(self.mesh_path).suffix.replace(".", "")`?